### PR TITLE
Add People/Esraa

### PR DESCRIPTION
**Pull request by ghost**
Adding myself to RADICAL's people page

### CODE REVIEW

**Comment by ghost** [comment]
@mturilli @iparask could you review it?

**Review by iparask — CHANGES_REQUESTED** [review]
Hello @EsraaAbdelmotteleb can you please remove the two converted markdown files?
Thank you

File: _people/esraa-abdelmotteleb.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: "Esraa Abdelmotteleb"
+collection: people
+avatar: esraa.jpg
+position: ms
+title: "Masters Student"
+permalink: /people/esraa-abdelmotteleb
+redirect_from:
```
Comment (iparask) [inline _people/esraa-abdelmotteleb.md]:
```suggestion
joined: 2020
```

File: _people/esraa-abdelmotteleb.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - "/esraa.html"
```
Comment (iparask) [inline _people/esraa-abdelmotteleb.md]:
```suggestion

```

File: _people/esraa-abdelmotteleb.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  - "people/esraa-abdelmotteleb.html"
```
Comment (iparask) [inline _people/esraa-abdelmotteleb.md]:
```suggestion

```

File: _people/esraa-abdelmotteleb.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  - "people/esraa-abdelmotteleb/index.html"
```
Comment (iparask) [inline _people/esraa-abdelmotteleb.md]:
```suggestion

```

File: _people/esraa-abdelmotteleb.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+name: "Esraa Abdelmotteleb"
+collection: people
+avatar: esraa.jpg
+position: ms
+title: "Masters Student"
```
Comment (iparask) [inline _people/esraa-abdelmotteleb.md]:
```suggestion
title: "Master Student"
```

**Review by iparask — CHANGES_REQUESTED** [review]
Hello @EsraaAbdelmotteleb,

thank you for removing the files, I was able to run the website and I have some additional suggestions. They are being provided as commit suggestions, so I think it should be a few clicks to get it there.

Thank you

**Comment by ghost** [comment]
Hi @iparask,

Thanks for the suggestions. I committed them. And thank you for reviewing this!

**Review by iparask — APPROVED** [review]

**Pull request merged**